main.go

```diff
@@ -26,6 +27,19 @@ var SlotsInEpoch = uint64(32)
 var UpdateValidatorsIntervalSlots = uint64(600)
 
 func main() {
+	//file is created if not exists, otherwise it appends errors to the existing file
+	//0666 means permisions to read and write to all users, but not execute
+	file, err := os.OpenFile("/oracle-data/oracleLogs.txt", os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
+	if err != nil {
+		log.Fatal("error opening or creating the oracleLogs.txt file: ", err)
+	}
+	defer file.Close()
+
+	// Create a MultiWriter with file and stdout
+	multiWriter := io.MultiWriter(file, os.Stdout)
+	// Set log output to the MultiWriter
+	log.SetOutput(multiWriter)
```

Comment:
just to confirm, does this logs everything? info, error, fatal, etc? assuming of course the logLevel is set to INFO.

Reply:
Yes, I have tested it using log level info and debug. Both printed info, warning and fatal logs